irohad/network/ordering_service.hpp

```diff
@@ -34,7 +35,9 @@ namespace iroha {
        * Transform model proposal to transport object and send to peers
        * @param proposal - object for propagation
        */
-      virtual void publishProposal(model::Proposal &&proposal) = 0;
+      virtual void publishProposal(
+          shared_model::detail::PolymorphicWrapper<
```

Comment:
It was a mistake to suggest using PolymorphicWrapper, since it does deep copy of object when a copy constructor is called. It is enough to use `shared_ptr` in component interfaces.

Reply:
Quite inconsistent interface then